### DISCUSSION TITLE
Lobe Chat Database version. GPT-4 Error

### DISCUSSION THREAD

**Discussion by harboot** [lobehub/lobehub#4238] [General Question] [answered]
![image](https://github.com/user-attachments/assets/0233bef3-49ef-43d1-aabd-a66d7e60bdae)

Can someone help me ?
Working fine with GPT-4mini, but error with other GPT model.

**Comment by SpeedupMaster** [comment, marked as answer]
does not have access to model "gpt-4o"
Your key likely doesn't have permission to access gpt-4o.

**Comment by harboot** [comment]
Yes, I just discovered the issue a few minutes ago. With the legacy API key, I don't need to specify the model used (there's no limit). However, in my current API project, I have to define the allowed models. Thank you for helping.
